### PR TITLE
579 adding preview command

### PR DESCRIPTION
**Pull request by Loumstar**
Added modules for the new preview_function() and unittests.
Preview function by default returns the incoming response.

### CODE REVIEW

File: app/preview_tests.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class TestPreviewFunction(unittest.TestCase):
+    """
+    TestCase Class used to test the algorithm.
+    ---
+    Tests are used here to check that the algorithm written
+    is working as it should.
+
+    It's best practise to write these tests first to get a
```
Comment (procaconsul) [inline app/preview_tests.py]:
```suggestion
    It's best practice to write these tests first to get a
```

**Review by procaconsul — APPROVED** [review]
LGTM

**Comment by Loumstar** [comment]
[Add preview command to Evaluation Functions](https://trello.com/c/2lOlowFa/579-add-preview-command-to-evaluation-functions)

**Comment by Loumstar** [comment]
[Add preview command to Evaluation Functions](https://trello.com/c/2lOlowFa/579-add-preview-command-to-evaluation-functions)

**Pull request merged**